src/Microsoft.AspNet.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -34,9 +37,8 @@ public DateHeaderValueManager()
             : this(
                   systemClock: new SystemClock(),
                   timeWithoutRequestsUntilIdle: TimeSpan.FromSeconds(10),
-                  timerInterval: TimeSpan.FromSeconds(1))
+                  timerInterval: TimeSpan.FromMilliseconds(200))
```

Comment:
@halter73 Sure, I'll change this to one second and submit another PR that sets it to 400ms.